### PR TITLE
Remove SensioDistributionBundle

### PR DESCRIPTION
**Pull request by yguedidi**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | no
| CHANGELOG.md  | no
| License       | MIT

Not supported in Symfony 4+

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,10 +9,8 @@
 !/var/sessions
 /var/sessions/*
 !var/sessions/.gitkeep
-!var/SymfonyRequirements.php
 /bin/*
 !/bin/console
-!/bin/symfony_requirements
```
Comment (yguedidi) [inline .gitignore:15]: seems it wasn't executed anywhere in the code base

File: composer.json [hunk 2]
```diff
@@ -121,10 +120,8 @@
     "scripts": {
         "post-cmd": [
             "Incenteev\\ParameterHandler\\ScriptHandler::buildParameters",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::buildBootstrap",
```
Comment (yguedidi) [inline composer.json:124]: no bootstrap file in Symfony 4

File: composer.json [hunk 3]
```diff
@@ -134,12 +131,6 @@
         ]
     },
     "extra": {
-        "symfony-app-dir": "app",
-        "symfony-bin-dir": "bin",
-        "symfony-var-dir": "var",
-        "symfony-web-dir": "web",
-        "symfony-tests-dir": "tests",
-        "symfony-assets-install": "relative",
```
Comment (yguedidi) [inline composer.json:142]: were used only by SensioDistributionBundle in the ScriptHandler

**Comment by yguedidi** [comment]
@j0k3r @Kdecherf @nicosomb the failing test seems not related to my change

**Comment by Kdecherf** [comment]
> @j0k3r @Kdecherf @nicosomb the failing tests seems not related to my change

Seeing that all failing tests are still failing after retrying them, it may suggest that it is

**Comment by yguedidi** [comment]
@Kdecherf thank you for the re-trigger! will investigate then

**Comment by yguedidi** [comment]
@Kdecherf fixed! it was long to investigate, but in the end it was quite simple... by default `Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::clearCache` doesn't warmup the cache, while just `cache:clear` does

**Review by nicosomb — DISMISSED** [review]

**Review by Kdecherf — DISMISSED** [review]
Looks good, there's a conflict with the composer.lock file however

**Comment by yguedidi** [comment]
updated!

**Review by Kdecherf — APPROVED** [review]

**Comment by yguedidi** [comment]
@j0k3r @Kdecherf @nicosomb congrats for the 2.5.0 release! :tada: 

I think this PR should be merge first, followed by #5810 (that I'll update ones this is merged).
Then, I'll make sure to update all my other PRs :slightly_smiling_face: 
And then the hard work on migrating to Symfony 4 will start! :muscle: 
I'll also open a PR to drop PHP 7.4 soon

**Review by j0k3r — APPROVED** [review]

**Pull request merged**